src/util/token.rs

```diff
@@ -120,8 +120,9 @@ secure_token_kind! {
     /// NEVER CHANGE THE PREFIX OF EXISTING TOKEN TYPES!!! Doing so will implicitly revoke all the
     /// tokens of that kind, distrupting production users.
     #[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
-    pub(crate) enum SecureTokenKind {
+    pub enum SecureTokenKind {
         Api => "cio", // Crates.IO
+        Session => "ses", // Session tokens.
```

Comment:
Not on the team anymore, but providing more context on the prefix: the goal of the prefix was to make sure secret scanning tooling could detect crates.io tokens easily, and that was the reason `cio` was present.

Not sure how the crates.io team wants the token to look, but maybe something like `bscio` (browser session crates.io)?

Reply:
Or maybe ciobs? That way the same crates.io token detection scan code could detect the browser token I think.

Reply:
That would conflict with `cio`, as `ciobs` is a valid `cio` token. That was poor planning when I originally created the prefix. There is a test preventing this behavior.

Reply:
how about `scio:` (session for crates.io), which has the advantage of being Latin for "I know," perfect for a bearer token. :-D

Reply:
Went with scio :)